docs/YIP/YIP-0001-price-precision.md

```diff
@@ -14,16 +14,36 @@ In the context of YN, we must have a network global configuration to make member
 
 ## Decision
 
+### Definitions
+
+**Price** - asset price in quote currency.
+
+**Quantity** - in trading, quantity usually refers to the number of units or shares
+of a particular security or asset that is being bought or sold.
+For instance, if an investor buys 100 shares of a company's stock, the quantity in this case is 100.
+
+**Value** - stands for USD market value, value is always in USD (for example for market BTC/ETH value will be in USD).
+
+| Number      | Round to |
+| ----------- | ----------- |
+| Price       | 18       |
+| Quantity    | 8        |
+| Value       | 2        |
```

Comment:
Add an explanation about what this table is.

Reply:
we dropped value at this point